pkg/services/order/order_fetcher.go

```diff
@@ -512,12 +513,18 @@ func branchFilter(branch *string, needsCounseling bool, ppmCloseoutGblocs bool)
 	}
 }
 
-func lastNameFilter(lastName *string) QueryOption {
+func customerNameFilter(name *string) QueryOption {
 	return func(query *pop.Query) {
-		if lastName != nil {
-			nameSearch := fmt.Sprintf("%s%%", *lastName)
-			query.Where("service_members.last_name ILIKE ?", nameSearch)
+		if name == nil {
+			return
 		}
+
+		// Remove "," that user may enter between names (displayed on frontend column)
+		nameQueryParam := *name
+		removeCharsRegex := regexp.MustCompile("[,]+")
+		nameQueryParam = removeCharsRegex.ReplaceAllString(nameQueryParam, "")
+		nameQueryParam = fmt.Sprintf("%%%s%%", nameQueryParam)
+		query.Where("((service_members.last_name || ' ' || service_members.first_name) || (service_members.first_name || ' ' || service_members.last_name)) ILIKE ?", nameQueryParam)
```

Comment:
Not necessary, but can this be simplified: 

```suggestion
		query.Where("(service_members.last_name || ' ' || service_members.first_name || ' ' || service_members.last_name) ILIKE ?", nameQueryParam)
```